App.tsx

```diff
@@ -1,8 +1,9 @@
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';
 import { Provider } from 'react-redux';
 import { store } from './reducers/store';
-import Player from './components/Player/player';
+import Player from './components/Player/Player';
```

Comment:
Looks like this will need to be lowercase, or the file uppercase

Reply:
Did you restart the server after pulling my changes?  It's working fine on my end